app/.gitignore

```diff
@@ -28,6 +28,8 @@
 /web/fileadmin/_processed_/**/*
 /web/uploads/**/*
 
+# remove the cssmapping files
+*.map
```

Comment:
i guess it makes sense to ignore them globally, because with every compiling, the map files are regenerated.